lib/S3FileSystem.js

```diff
@@ -46,14 +46,14 @@ class S3FileSystem {
      * @param ImageData image
      * @return Promise
      */
-    putObject(image) {
+    putObject(image, options) {
         const params = {
             Bucket:       image.bucketName,
             Key:          image.fileName,
             Body:         image.data,
             Metadata:     { "img-processed": "true" },
             ContentType:  image.headers.ContentType,
-            CacheControl: image.headers.CacheControl,
+            CacheControl: (options.cacheControl !== undefined) ? options.cacheControl : image.headers.CacheControl,
```

Comment:
`options.cacheControl || image.headers.CacheControl` would be better.

Reply:
Like I said above, if the user wants to new file to *not* have CacheControl, even though the original file had one, using `||` would prevent it